tensorboard/backend/application.py

```diff
@@ -151,10 +151,26 @@ def TensorBoardWSGIApp(logdir, plugins, multiplexer, reload_interval,
     ValueError: If something is wrong with the plugin configuration.
   """
   path_to_run = parse_event_files_spec(logdir)
+  thread_name = 'Multiplexer Reloader'
```

Comment:
nit, but I think just 'Reloader' was probably enough (or even 'Reload') - this gets printed in every single log line from this thread, so probably best to keep it succinct.